twoliter/embedded/build.Dockerfile

```diff
@@ -166,9 +169,12 @@ RUN --mount=source=.cargo,target=/home/builder/.cargo \
     --mount=type=cache,target=/home/builder/.cache,from=cache,source=/cache \
     --mount=type=cache,target=/home/builder/rpmbuild/BUILD/sources/models/src/variant,from=variantcache,source=/variantcache \
     --mount=source=sources,target=/home/builder/rpmbuild/BUILD/sources \
+    # '-' is an illegal char in RPM Release field; -dirty commits become _dirty
+    BUILD_ID=$(echo ${BUILD_ID} | tr '-' '_') ; \
     rpmbuild -ba --clean \
       --undefine _auto_set_build_flags \
       --define "_target_cpu ${ARCH}" \
+      --define "dist ${BUILD_ID_TIMESTAMP}_${BUILD_ID}" \
```

Comment:
`-` instead of `_` is more conventional. Also may as well add the `br1` distro tag if we are going to use this field.
```suggestion
      --define "dist ${BUILD_ID_TIMESTAMP}-${BUILD_ID}.br1" \
```